### PR TITLE
Add Paths for KZ test map

### PR DESCRIPTION
**Pull request by bytsbyts**

### CODE REVIEW

File: Assets/KZpath1.cs
Comment (matias-gonz) [inline Assets/KZpath1.cs:1]: Remove file

**Review by matias-gonz — CHANGES_REQUESTED** [review]

File: KZpath1.cs
Comment (matias-gonz) [inline KZpath1.cs:1]: Remove file

**Review by matias-gonz — CHANGES_REQUESTED** [review]

**Comment by bytsbyts** [comment]
Removed it,sorry for the late operation



---- Replied Message ----
| From | Matías Ignacio ***@***.***> |
| Date | 06/17/2024 10:09 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [matias-gonz/behind-his-eyes] add the path name of enemy controlling (PR #15) |

@matias-gonz requested changes on this pull request.

On KZpath1.cs:

Remove file

**Comment by bytsbyts** [comment]
I pushed new changes, can you take a look at it

File: Assets/Resources/.DS_Store
Comment (matias-gonz) [inline Assets/Resources/.DS_Store:1]: Shouldn't upload these

**Review by matias-gonz — CHANGES_REQUESTED** [review]

**Comment by matias-gonz** [comment]
Deprecated

**Pull request closed without merging**